client/src/charts/wrapped_nivo/WrappedNivoTreemap/WrappedNivoTreemap.stories.js

```diff
@@ -0,0 +1,108 @@
+import { scaleOrdinal } from "d3-scale";
+import _ from "lodash";
+import React from "react";
+
+import { newIBLightCategoryColors } from "src/core/color_schemes";
+import { formats } from "src/core/format";
+
+import { WrappedNivoTreemap } from "./WrappedNivoTreemap";
+
+export default {
+  title: "charts/WrappedNivoTreemap",
+  component: WrappedNivoTreemap,
+};
+
+const graph_data_values = [
+  32432432,
+  8324234,
+  9456354,
+  5795646,
+  23746845,
+  8723476,
+  23435435,
+  324323,
+];
+
+const graph_data_total_amt = _.sum(graph_data_values);
+
+const graph_data = {
+  children: [
+    {
+      desc: "Description 1",
+      id: 0,
+      total: graph_data_total_amt,
+      value: graph_data_values[0],
+    },
+    {
+      desc: "Description 2",
+      id: 1,
+      total: graph_data_total_amt,
+      value: graph_data_values[1],
+    },
+    {
+      desc: "Description 3",
+      id: 2,
+      total: graph_data_total_amt,
+      value: graph_data_values[2],
+    },
+    {
+      desc: "Description 4",
+      id: 3,
+      total: graph_data_total_amt,
+      value: graph_data_values[3],
+    },
+    {
+      desc: "Description 5",
+      id: 4,
+      total: graph_data_total_amt,
+      value: graph_data_values[4],
+    },
+    {
+      desc: "Description 6",
+      id: 5,
+      total: graph_data_total_amt,
+      value: graph_data_values[5],
+    },
+    {
+      desc: "Description 7",
+      id: 6,
+      total: graph_data_total_amt,
+      value: graph_data_values[6],
+    },
+    {
+      desc: "Description 8",
+      id: 7,
+      total: graph_data_total_amt,
+      value: graph_data_values[7],
+    },
+  ],
+  color: "white",
+  name: "root",
+};
+
+const sorted_children = _.reverse(
+  _.sortBy(graph_data.children, function (o) {
+    return o.value;
+  })
+);
+
+const domain = [];
+_.forEach(sorted_children, function (child) {
+  _.concat(domain, child.value);
+});
```

Comment:
Gotta warn you before the for loop police show up ([it's in our conventions](https://github.com/TBS-EACPD/infobase/blob/master/docs/getting-started/js-tips-style-guide.md#javascript-coding-conventions))
[First google search on why not to use for loops
](https://medium.com/python-pandemonium/never-write-for-loops-again-91a5a4c84baf)

Before the "actual" fix though, `domain` will be an empty array for 2 different reasons.
1. `_.concat` is **not** changing the contents inside `domain`. (No mutation). So in order for this `concat` logic to work, you'd need to do `domain = _.concat(domain, child.value)`. Which brings me to 2nd reason
2. Since you declared `domain` as **`const`**,  you'll get an error because `domain` is constant! So to make this code work, you need
```
let domain = [];
_.forEach(sorted_children, (child) => {
  domain = _.concat(domain, child.value)
}
```

But as I mentioned, we wanna try to avoid for loops as much as possible. So instead, you could simply do (both equivalent):
`_.map(sorted_children, (child) => child.value)`
`_.map(sorted_children, "value")`